File: app/src/main/java/com/tregula/graedukacyjna/view/ScoreView.java
package com.tregula.graedukacyjna.view;

import android.content.Context;
import android.util.AttributeSet;
import android.widget.LinearLayout;
import android.widget.TextView;

import com.tregula.graedukacyjna.R;

import androidx.annotation.Nullable;

public class ScoreView extends LinearLayout {

    private TextView score_textView;

    public ScoreView(Context context) {
        super(context);
        init(context);
    }

    public ScoreView(Context context, @Nullable AttributeSet attrs) {
        super(context, attrs);
        init(context);
    }

    public ScoreView(Context context, @Nullable AttributeSet attrs, int defStyleAttr) {
        super(context, attrs, defStyleAttr);
        init(context);
    }

    public void init (Context context)
    {
        inflate( context, R.layout.score_view,this);
        score_textView = findViewById(R.id.score_textView);
    }
}

File: app/src/main/java/com/tregula/graedukacyjna/view/ScoreView.kt
package com.tregula.graedukacyjna.view

import android.content.Context
import android.util.AttributeSet
import android.view.View
import android.widget.RelativeLayout
import android.widget.TextView
import androidx.appcompat.widget.Toolbar
import com.tregula.graedukacyjna.R

class ScoreView @JvmOverloads constructor(
        context: Context, attrs: AttributeSet? = null, defStyleAttr: Int = 0
) : RelativeLayout(context, attrs, defStyleAttr) {

    var onNavigateBack: () -> Unit = {}

    private val score: TextView

    init {
        View.inflate(context, R.layout.score_view, this)
        score = findViewById(R.id.score_label)

        val toolbar: Toolbar = findViewById(R.id.toolbar)
        toolbar.setNavigationOnClickListener {
            onNavigateBack()
        }
    }

    fun setScore(scoreResult: Int) {
        score.text = context.getString(R.string.score, scoreResult.toString())
    }
}
